src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/InProcess/RoslynSettingsFileSynchronizerTests.cs

```diff
@@ -194,6 +194,46 @@ public async Task UpdateFileStorage_IssuesAreConvertedAndFiltered()
             actualSuppressions[1].RoslynRuleId.Should().Be("S222");
         }
 
+        [TestMethod]
```

Comment:
This test is almost a complete copy from the test above. The only change is that it tests if suppressed issues get filtered